scene/gui/text_edit.cpp

```diff
@@ -825,6 +834,9 @@ void TextEdit::_notification(int p_what) {
 			// get the highlighted words
 			String highlighted_text = get_selection_text();
 
+			// check if highlighted words contains only whitespaces (tabs or spaces)
+			bool only_whitespaces_highlighted = _contains_only_whitespaces(highlighted_text);
```

Comment:
You could avoid adding a new method by using `highlighted_text.strip_edges() == String()`.

Reply:
fixed